manager/controllers/app/fybrikapplication_controller_unit_test.go

```diff
@@ -585,6 +585,91 @@ func TestMultipleDatasets(t *testing.T) {
 	g.Expect(plotter.Spec.Flows[1].SubFlows).To(gomega.HaveLen(2))
 }
 
+// XXXXX
+// Assumptions on response from connectors:
```

Comment:
These and inner comments do not reflect this test

Reply:
fixed